examples/computer_vision/detr_coco_pytorch/startup-hook.sh

```diff
@@ -7,9 +7,10 @@ mv annotations/instances_train2017.json /tmp
 mv annotations/instances_val2017.json /tmp
 
 git clone https://github.com/facebookresearch/detr.git
-cd detr && git reset --hard 4e1a9281bc5621dcd65f3438631de25e255c4269 && cd ..
+cd detr && git reset --hard 4e1a9281bc5621dcd65f3438631de25e255c4269 
```

Comment:
Is this a fix for the previous version? I'm assuming that the `cd ..` meant that the python file couldn't be found since we exited its parent directory?

Reply:
Checked the error logs, and this makes sense. Resolved.